src/pages/home/report/ReportActionsList.tsx

```diff
@@ -210,12 +210,9 @@ function ReportActionsList({
         [sortedReportActions, isOffline],
     );
 
-    // whisper action doesn't affect lastVisibleActionCreated, so we should not take it into account while checking if there is the newest report action
-    const newestVisibleReportAction = useMemo(() => sortedVisibleReportActions.find((item) => !ReportActionsUtils.isWhisperAction(item)) ?? null, [sortedVisibleReportActions]);
```

Comment:
> Looks like we will reintroduce https://github.com/Expensify/App/issues/41188

@parasharrajat 

I believe it's a back-end bug. After we start s split bill, BE doesn't return the updated `lastVisibleActionCreated` of the report but when `OpenReport` API is called, we can see the `lastVisibleActionCreated` is returned as the `created` of split whisper action above.


<img width="1512" alt="Screenshot 2024-06-05 at 15 03 54" src="https://github.com/Expensify/App/assets/161821005/427cf17a-395c-46a6-9182-9aa9c10ceeca">

So I removed this change from this PR. If not when we reload this report and send any action like message, expense, ... the chat will not scroll to the bottom

Reply:
Is this for the same account that created the split bill request? If so, we should use the whisper `created`, no?

Reply:
Commented in [Slack](https://expensify.slack.com/archives/C05R2V5GM1S/p1717679355054139?thread_ts=1716297653.378619&cid=C05R2V5GM1S)

Reply:
Following along because I'm the BZ team member on https://github.com/Expensify/App/issues/41188. Do you all feel that a regression was introduced in that issue? It's due to be paid out, but I will hold on that if there's a regression.

Reply:
Awesome. Thanks, @parasharrajat!